### ISSUE TITLE
Cant login with new version

### ISSUE THREAD

**Issue by eggsaladmofo** [xyzz/acquisition#481]
Rip tried everything

**Comment by sockenklaus** [comment]
Had the same problem because of an expired session ID. Did you try to relog with email / password?

**Comment by eggsaladmofo** [comment]
yes it does not recognize session or my actual account info, ive tried for days now and no trading at all, it worked with the old version flawlessly

**Comment by nocturnal2** [comment]
Same here - just downloaded the latest version and it would not accept my email and password

**Comment by Hexor6T** [comment]
Doesn't accept my email & password either. Rip my trading
